com.haysell.pos.json

```diff
@@ -0,0 +1,39 @@
+{
+    "id":"com.haysell.pos",
+    "runtime":"org.kde.Platform",
+    "runtime-version":"5.15-23.08",
+    "sdk":"org.kde.Sdk",
+    "command":"haysell-pos",
+    "finish-args":[
+      "--share=ipc",
+      "--share=network",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--device=dri",
+      "--socket=pulseaudio",
+      "--filesystem=host"
+    ],
+    "cleanup":[
+      "/include",
+      "*.la",
+      "*.a"
+    ],
+    "modules":[
+      {
+        "name":"haysell-pos",
+        "buildsystem":"cmake-ninja",
+        "sources":[
+          {
+            "type":"git",
+            "url":"https://github.com/Delphi251189/haysell-pos.git",
+            "commit" : "fa8490b536bee11345e30392a3e6de8f0e8b4ea7"
+          }
+        ],
+        "post-install": [
+            "cp -f flatpak/flatpak/com.haysell.pos.metainfo.xml /app/share",
+            "cp -f flatpak/com.haysell.pos.desktop /app/share",
+            "cp -f flatpak/com.haysell.pos.png /app/share"
```

Comment:
Where did you get this paths from? They aren't correct. https://docs.flatpak.org/en/latest/conventions.html#metainfo-files

Reply:
I already understood...
Paths from upstream repo, but first searches in google says, that they must be `declared` as `source`s
![image](https://github.com/user-attachments/assets/385eabfc-e8c8-4db7-889e-275f19a876aa)

Reply:
is there any **short** way to **copy files from src directory** in `post-install` stage???

Reply:
I'm am taking about where you are installing them, `/app/share` is not correct. See the docs I linked.

Reply:
Also, first look at other projects shows, that those 3 files must be located in different directories, like `/app/share/icons/hicolor/` , `/app/share/appdata/` and `/app/share/applications` ( instead of the same /app/share )

Is this correct?

Reply:
and everything works just fine ( launcher icon)

Reply:
> Is it ok to just call mkdir -p foo/bar/baz ?

`install -D` can do it in one command.